example-deployment/keycloak-tools/bootstrap-keycloak.js

```diff
@@ -97,7 +97,69 @@ function authHeaders(token) {
         eventsListeners: ['jsonlog_event_listener'],
       },
     );
+
+    // create audience client scope
+    await request(
+      `${baseUrl}/admin/realms/rmf-web/client-scopes`,
+      {
+        method: 'POST',
+        headers: authHeaders(token),
+      },
+      {
+        name: 'dashboard',
+	protocol: 'openid-connect',
+	description: 'dashboard scope',
+	protocolMappers: [
+	  {
+	    config: {
+	      "access.token.claim": "true",
+	      "id.token.claim": "false",
+	      "included.client.audience": "dashboard"
+	    },
+	    name: 'rmf-audience',
+ 	    protocol: 'openid-connect',
+  	    protocolMapper: 'oidc-audience-mapper'
+	  }
+	]
+      },
+    );
+
+    // get dashboard id ( not clientid )
+    resp = await request(`${baseUrl}/admin/realms/rmf-web/clients`, {
+      method: 'GET',
+      headers: authHeaders(token),
+    });
+
+    const client_array = JSON.parse(resp.body);
+    const dashboard_id = client_array.filter(function(item){
+	    return item.clientId == "dashboard";
```

Comment:
use triple equal to avoid weird type errors.

Reply:
handled here: https://github.com/open-rmf/rmf-web/pull/350/commits/1bf2a983486f5400bde48a0a249288f2d2754dc5